cmd/create.go

```diff
@@ -239,7 +239,9 @@ func newCreateConfig(cmd *cobra.Command, args []string, newClient ClientFactory)
 	// it is still available as an environment variable.
 	repositoriesPath = os.Getenv("FUNC_REPOSITORIES_PATH")
 	if repositoriesPath == "" { // if no env var provided
-		repositoriesPath = fn.New().RepositoriesPath() // use ~/.config/func/repositories
+		client, done := newClient(ClientConfig{})
```

Comment:
Can't use the default client here, because we want to (indirectly) use the path provided to Client via `config.RepositoriesPath`